src/main/java/org/osiam/Osiam.java

```diff
@@ -2,20 +2,16 @@
 
 import com.google.common.collect.ImmutableMap;
 import com.ryantenney.metrics.spring.config.annotation.EnableMetrics;
-import com.zaxxer.hikari.HikariConfig;
-import com.zaxxer.hikari.HikariDataSource;
+import com.zaxxer.hikari.*;
```

Comment:
I'm not fundamentally opposed to wildcard  imports, even though I'm not a big fan of them. But please don't commit files with this PR that don't have any changes other than import collapses.